### PR TITLE
Complete mitigation section

### PR DESCRIPTION
**Pull request by jyao1**
Fix https://github.com/riscv-non-isa/riscv-ap-tee-io/issues/16

### CODE REVIEW

File: specification/05-security_model.adoc [hunk 1]
```diff
@@ -384,8 +407,11 @@ from the following adversaries:
     eavesdrop or tamper with the TVM confidential data.
 
 5+^| **Mitigations**
-5+a| TBD
+5+a| The `CoVE-IO-T009` threat can be addressed as follows:
 
+    - The TSM must guarantee the DMA translation table for one TDI can only
+      access the corresponding TVM.
+    - The DSM must guarantee the DMA request uses IDE TLP with T-bit set.
```
Comment (sameo) [inline specification/05-security_model.adoc:414]:
```suggestion
    - The RDSM must guarantee that a DMA transaction from one TDI is translated by an IOMMU instance controlled by a TSM that manages the TVM to which the TDI is bound to.
    - The TSM must guarantee that the DMA translation table for one TDI can only
      access the corresponding TVM.
    - The DSM must guarantee that the DMA request uses IDE TLP with T-bit set.
```

File: specification/05-security_model.adoc [hunk 2]
```diff
@@ -414,31 +440,10 @@ from the following adversaries:
...
-
-5+^| **Description**
-5+| A privileged host software component resets or powers down an assigned TDI
-    or its physical device, while the TDI is assigned to a TVM. +
-    The TVM is no longer able to directly access its assigned TDI.
-
-5+^| **Mitigations**
-5+a| TBD
+    - The TVM must accept the DMA translation table explictely.
+    - The TSM must not enable DMA translation table until the TVM accepts it.
```
Comment (sameo) [inline specification/05-security_model.adoc:446]: I don't think the TVM should or can accept the DMA translation tables.

```suggestion
    - The RDSM must guarantee that a DMA transaction from one TDI is translated by an IOMMU instance controlled by a TSM that manages the TVM to which the TDI is bound to.
    - The TSM must guarantee that the DMA translation table for a TDI under its control is consistent with the G-stage tables for the TVM the TDI is bound to.
    - The TSM must not enable DMA translation table until the TVM accepts the TDI.
```

File: specification/05-security_model.adoc (hunk 1 above)
Comment (jyao1) [reply specification/05-security_model.adoc]: Updated.

File: specification/05-security_model.adoc (hunk 2 above)
Comment (jyao1) [reply specification/05-security_model.adoc]: > I don't think the TVM should or can accept the DMA translation tables.

For GPA->HPA, right, TVM cannot do anything.
But if DMA translation table also has GVA->GPA, then TVM needs to provide input.

**Review by sameo — APPROVED** [review]

**Pull request merged**